src/Console/ConfigurationResolver.php

```diff
@@ -702,13 +702,32 @@ private function validateRules(array $rules)
         );
 
         if (count($unknownFixers)) {
-            throw new InvalidConfigurationException(sprintf(
-                'The rules contain unknown fixers (%s).',
-                implode(', ', $unknownFixers)
-            ));
+            $message = 'The rules contain unknown fixers (';
+            foreach ($unknownFixers as $unknownFixer) {
+                $alternative = $this->getAlternativeRule($unknownFixer, $availableFixers);
+                $message .= sprintf(
+                    '"%s"%s, ',
+                    $unknownFixer,
+                    null === $alternative ? '' : ' maybe use "'.$alternative.'"'
+                );
+            }
+
+            throw new InvalidConfigurationException(substr($message, 0, -2).').');
         }
     }
 
+    private function getAlternativeRule($unknownFixer, array $availableFixers)
```

Comment:
DRY is violated

Reply:
Please drop this method and use `WordMatcher` instead.
(branch rebase is needed)